apps/nextjs/src/middleware.ts

```diff
@@ -51,7 +51,7 @@ const nextMiddleware: NextMiddleware = async (request, event) => {
 
   try {
     response = await authMiddleware(request, event);
-    response = addCspHeaders(response, request, cspConfig);
+    response = await addCspHeaders(response, request, cspConfig);
```

Comment:
It doesn't look like this returns a promise? I don't ever see the eslint rule about missing an await on a promise. If we get that working, we would be alerted the function started to be async and needed to be awaited

ps: Obviously this is for another PR when this one is stable and ready to merge